### PR TITLE
Add call page object

### PR DESCRIPTION
**Pull request by ivy-lli**

### CODE REVIEW

File: integrations/standalone/tests/integration/parts/call.ts [hunk 1]
```diff
@@ -1,58 +1,68 @@
...
+    this.code = part.scriptArea();
   }
-  async fill({ page }: Part) {
-    await ComboboxUtil.select(page, this.selectValue, this.selectLabel);
-    await TableUtil.fillExpression(page, 2, '"test"');
-    await CodeEditorUtil.fill(page, 'ivy.log.info("hi");');
+
+  async fill() {
+    await this.call.choose(this.selectValue);
+    await this.mapping.row(2).column(2).edit('"test"');
```
Comment (weissreto) [inline integrations/standalone/tests/integration/parts/call.ts:21]: use fill,
edit does not close script dialog

File: integrations/standalone/tests/integration/parts/call.ts [hunk 2]
```diff
@@ -1,58 +1,68 @@
...
+    await this.code.clear();
   }
-  async assertClear({ page }: Part) {
-    await ComboboxUtil.assertSelect(page, this.assertSelectValue, this.selectLabel);
-    await TableUtil.assertRow(page, 2, ['']);
-    await CodeEditorUtil.assertValue(page, '');
+
+  async assertClear() {
+    await this.call.expectValue(this.assertSelectValue);
+    await this.mapping.row(2).column(2).expectValue('');
```
Comment (weissreto) [inline integrations/standalone/tests/integration/parts/call.ts:38]: this.mapping.expectEmpty ?

File: integrations/standalone/tests/pageobjects/Composite.ts [hunk 3]
```diff
@@ -28,6 +29,10 @@ export abstract class Composite {
     return new Select(this.page, this.locator, label);
   }
 
+  combobox(label: string) {
```
Comment (weissreto) [inline integrations/standalone/tests/pageobjects/Composite.ts:32]: There is already a select widget. What is the difference between combo and select?

**Review by weissreto — CHANGES_REQUESTED** [review]

File: integrations/standalone/tests/pageobjects/Composite.ts (hunk 3 above)
Comment (ivy-lli) [reply integrations/standalone/tests/pageobjects/Composite.ts:32]: yes :)

**Review by weissreto — APPROVED** [review]

**Pull request merged**